libraries/provider_hab_package.rb

```diff
@@ -124,7 +124,7 @@ def depot_package(name, version = nil)
           begin
             origin, pkg_name = name.split('/')
             name_version = [pkg_name, version].compact.join('/').squeeze('/').chomp('/').sub(%r{^\/}, '')
-            url = "#{new_resource.bldr_url.chomp('/')}/v1/depot/channels/#{origin}/#{new_resource.channel}/pkgs/#{name_version}"
+            url = "#{new_resource.bldr_url.chomp('/')}/depot/channels/#{origin}/#{new_resource.channel}/pkgs/#{name_version}"
```

Comment:
This would be a breaking change for anyone currently using the cookbook.  We should do this instead:
```
url = if new_resource.bldr_url.include?('/v1/')
        "#{new_resource.bldr_url.chomp('/')}/depot/channels/#{origin}/#{new_resource.channel}/pkgs/#{name_version}"
      else
        "#{new_resource.bldr_url.chomp('/')}/v1/depot/channels/#{origin}/#{new_resource.channel}/pkgs/#{name_version}"
      end
```